code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1955,3 +1955,24 @@ Eyes need to have significantly high darksight to shine unless the mob has the X
 			to_chat(usr, "[src] does not have any stored infomation!")
 	else
 		to_chat(usr, "OOC Metadata is not supported by this server!")
+
+/mob/living/carbon/human/ObjBump(A)
+	if(HAS_TRAIT(src, TRAIT_FORCE_DOORS))
+		if(istype(A, /obj/machinery/door/airlock))
+			var/obj/machinery/door/airlock/AL = A
+			if(AL.operating)
+				return
+			if(AL.welded)
+				to_chat(src, "<span class='warning'>The door is welded.</span>")
+				return
+			else if(AL.locked)
```

Comment:
No need for the `else`, since you're returning:
```suggestion
			if(AL.welded)
				to_chat(src, "<span class='warning'>The door is welded.</span>")
				return
			if(AL.locked)
```